pywbemtools/pywbemcli/_pywbemcli_operations.py

```diff
@@ -316,14 +318,23 @@ def build_repository(conn, server, file_path_list, verbose):
                 try:
                     # Displays any MOFParseError already
                     conn.compile_mof_file(file_path)
-                except MOFParseError:
+                except Error as er:
                     # Abort the entire pywbemcli command because the
                     # MOF compilation might have caused inconsistencies in the
                     # mock repository.
-                    click.echo(
-                        "Mock MOF file '{}' failed compiling (see above)".
-                        format(file_path),
-                        err=True)
+
+                    if not PYWBEM_VERSION.startswith("0"):
```

Comment:
I would prefer a more robust and also more self-explanatory test for the pywbem version being at least 1.0.0:
```
import pywbem
import packaging.version

PYWBEM_VERSION = packaging.version.parse(pywbem.__version__)

. . .
if PYWBEM_VERSION.release >= (1, 0, 0):
```
That allows also checking whether it is a beta version or dev version, if needed:
```
if PYWBEM_VERSION.is_devrelease:  # a dev release
if PYWBEM_VERSION.is_prerelease:  # an alpha or beta or rc release
```

Reply:
Changed code.  Note that I also changed code in tests to use the parse module.

Reply:
OK